### ISSUE TITLE
./ot-recorder: No such file or directory

### ISSUE THREAD

**Issue by nsalvatore96** [owntracks/recorder#281]
I don't own a Linux computer so I created an Amazon EC2 instance, free tier, and installed Debian 9.

Pasted the installation commands:
```
curl http://repo.owntracks.org/repo.owntracks.org.gpg.key | sudo apt-key add -
echo "deb  http://repo.owntracks.org/debian stretch main" | sudo tee /etc/apt/sources.list.d/owntracks.list > /dev/null
sudo apt-get update
sudo apt-get install ot-recorder
```
But when trying to launch the recorder using " ./ot-recorder 'owntracks/#' ", I get the no such file or directory error...

![screenshot 1](https://user-images.githubusercontent.com/30666582/51407174-e1fc3980-1b39-11e9-8be3-549d4e928e29.JPG)
![screenshot 2](https://user-images.githubusercontent.com/30666582/51407173-e1fc3980-1b39-11e9-9f92-dc7191570efc.JPG)

**Comment by jpmens** [comment]
You'll find the package has installed the program and certain supporting files in the locations compiled in to the programs. The Recorder itself is in 

```
/usr/sbin/ot-recorder
```

on Debian, so launching

```
$ /usr/sbin/ot-recorder
```

ought to get you further. However, please check first that it isn't already running:

```
$ ps ax|grep recorder
```

as the Debian package launches it (I think)

**Comment by nsalvatore96** [comment]
> You'll find the package has installed the program and certain supporting files in the locations compiled in to the programs. The Recorder itself is in
> 
> ```
> /usr/sbin/ot-recorder
> ```
> on Debian, so launching
> 
> ```
> $ /usr/sbin/ot-recorder
> ```
> ought to get you further. However, please check first that it isn't already running:
> 
> ```
> $ ps ax|grep recorder
> ```
> as the Debian package launches it (I think)

You were right, thanks. 

Now this is the first time I use Linux and the command line... I can't figure out how to configure Recorder. My OwnTracks app is running on my phone and it is connected to my MQTT server but I don't know how to tell Recorder what the MQTT server information is. Server name, user, password, port, etc.

Do you mind telling me exactly what to type in the command line? THANK YOU!!

**Comment by jpmens** [comment]
You will have to study our documentation. The Recorder itself is [documented here](https://github.com/owntracks/recorder/blob/master/README.md), and our [OwnTracks Booket](https://owntracks.org/booklet/) also contains a lot of information, particularly on MQTT and how OwnTracks uses it.

If you are completely new to Linux I'd like to reccomend you first try and get to know some basic file system commands and also how to use a text editory (e.g. `joe` or `pico` or `nano`) which will allow you to enter text to configure the Recorder (but also other aspects of your Linux system).

**Comment by jpmens** [comment]
The two environment variables `$OTR_USER` and `$OTR_PASS` can be used for that.

**Comment by jpmens** [comment]
Please read our documentation; we spent a lot of effort on it.

**Comment by nsalvatore96** [comment]
Trust me I am putting a lot of effort on trying to understand the documentation!! The problem is my field of study is economics, not computer science... I just like technology a lot and spend my free time learning and trying to apply it to my everyday life.

When I come to this issue it's because I really need somebody's help. Thanks.

**Comment by nsalvatore96** [comment]
I finally got it working!!! Now my last step is being able to access the web interface from "outside".
I can access by connecting to my Amazon Instance (Debian 9 Stretch) via SSH and opening a web browser and typing 127.0.0.1:8083/index.html but when I try connecting from my local computer to PUBLIC_IP:8083/index.html, I get connection refused. 

I figured I maybe had to set OTR_HTTPHOST to my private or public IP instead of localhost but none of those worked. Changed that in the config but still when I launch ot-recorder there's a line that says "HTTP listener started on 127.0.0.1:8083"... Maybe there's the problem?

I also made sure my port was forwarded.

Thanks
